internal/pkg/term/selector/selector_test.go

```diff
@@ -1226,23 +1233,23 @@ func TestWorkspaceSelect_Dockerfile(t *testing.T) {
 			mockPrompt: func(m *mocks.MockPrompter) {
 				m.EXPECT().SelectOne(
 					gomock.Any(), gomock.Any(),
-					gomock.Eq([]string{
-						"./Dockerfile",
-						"backend/Dockerfile",
-						"frontend/Dockerfile",
-						"Use an existing image instead",
-					}),
+					gomock.Eq(dockerfileOptions),
 					gomock.Any(),
 				).Return("frontend/Dockerfile", nil)
 			},
 			wantedErr:        nil,
 			wantedDockerfile: "frontend/Dockerfile",
 		},
-		"prompts user for custom path if fail to find Dockerfiles": {
+		"prompts user for custom path": {
```

Comment:
Can we make sure there's a test for the case where no dockerfiles are found and we only ask "Input custom dockerfile" and "Select existing image?"

Reply:
:+1:

Reply:
done!